packages/smooth_app/lib/pages/product_list_user_dialog_helper.dart

```diff
@@ -215,8 +215,9 @@ class ProductListUserDialogHelper {
     final bool? deleted = await showDialog<bool>(
       context: context,
       builder: (final BuildContext context) => SmoothAlertDialog(
-        title: 'Delete list?',
```

Comment:
Why are we removing the title?

Reply:
The point of this PR is to put an appropriate label:
* in this particular case, the title was no localized
* we need to be able to say "do you want to clear THIS list?" and/or "do you want to clear the HISTORY?"
* we could keep this title, but that would mean that 1. we should localize it and 2. it would be kind of duplicate with the more detailed message "clear this list" / "clear the history"

Reply:
In my opinion having a title, even a generic one, could make the decision faster. As a user I'd first like to read "Are you sure?" and then if I need an extended explanation I'm reading the body.

It's a personal preference though, nothing blocking so feel free to merge!